docs/docsite/rst/presentations/ansible-testing.rst

```diff
@@ -0,0 +1,426 @@
+Ansible Testing
+===============
+
+
+Current as of Ansible 2.3 (May 2017)
+
+Who am I?
+=========
+
+John Barker
+
+* Principal Engineer, Ansible by Red Hat
+* Hats: Ansible Networking, QA and community
+* Freenode/GitHub: `gundalow <https://github.com/gundalow>`_
+
+.. container:: handout
+
+   * Based in the UK
+   * ~ 1 year
+
+Ansible Testing
+===============
+
+* History of testing Ansible
+* How `you` can develop tests
+* Where to find out more
+
+.. container:: handout
+
+   * Set the scene
+
+     * Quick taster, give you enough to know what's possible and point you in the right direction
+     * Important: Reduce regression, help support Python 3
+
+   * Tailor content:
+
+     * Q: Show of hands for version, devel, 2.3, -> 1.9? - upgrade!
+     * Q: Who's raised a PR, should be getting easier on each release, especially modules
```

Comment:
Maybe mention that in order to improve existing modules/documentation, this can be done through the Github web interface (simply browse to the file and press edit). No prior knowledge of Git, or specific access is needed to start contributing !